### DISCUSSION TITLE
Option to remove or possibly configure the amount of padding used around the edges of the editor

### DISCUSSION THREAD

**Discussion by oku32** [helix-editor/helix#8187] [General]
I add padding to my terminal emulator as I think it looks a lot prettier in normal use. This requires a transparent background to look right, which is perfect, as Helix provides this functionality.

Helix adds padding around the edges of the editor, which adds to this amount. I would like it to sit in-line with tmux, which doesn't include padding by default. It also affects the gutter, as expected.

I would like to be able to remove this padding, and then I assume that being able to set a custom padding amount might then be desired by other users? 

*Default Helix padding*
![Helix default padding](https://github.com/helix-editor/helix/assets/125210745/6c90943d-6464-402a-8c48-81851aed5bc8)

*Helix without padding*
![Helix without padding](https://github.com/helix-editor/helix/assets/125210745/b03bfcc2-0733-43e6-88b9-aca7e7bf6d64)

**Comment by kirawi** [comment]
There is no padding. If the window isn't big enough to fill the entire space with cells, it'll add an unusable margin that Helix can't do anything about.

Edit: Ah wait I misunderstod the images.

This is fixed by https://github.com/helix-editor/helix/pull/7310
